libs/libc/unistd/lib_getcwd.c

```diff
@@ -63,6 +63,12 @@
  *   symbolic links. The 'size' argument is the size in bytes of the
  *   character array pointed to by the 'buf' argument.
  *
+ *   As an extension to the POSIX.1-2001 standard, getcwd() allocates
+ *   the buffer dynamically using malloc if buf is NULL.  In this case,
+ *   the allocated buffer has the length size  unless size is zero, when buf
```

Comment:
missing comma?

Reply:
what?

Reply:
the allocated buffer has the length size  unless size is zero, when buf
the allocated buffer has the length size, unless size is zero, when buf

Reply:
Or remove the extra space?

Reply:
done！